disclosure/urls.py

```diff
@@ -15,13 +13,16 @@
     url(r'^static/(?P<path>.*)$', 'django.views.static.serve', {
         'document_root': settings.STATIC_ROOT,
         'show_indexes': True}),
+
+    url(r'', include('ballot.urls')),
+    url(r'', include('election_day.urls')),
+    url(r'', include('finance.urls')),
+    # url(r'', include('locality.urls')),  # empty
+
     url(r'^search/', views.search_view),
     url(r'^locations/(?P<locality_id>[0-9]+)$', views.location_view),
-    url(r'^docs/', include('rest_framework_swagger.urls')),
-)
-
-# Register all API Viewsets:
-api = SimpleRouter()
-api.register(r'contributions', views.Contribution)
-api.register(r'elections', election_day.views.ElectionDay)
-urlpatterns += api.urls
+    url(r'^committee/(?P<committee_id>[0-9]+)$', views.committee_view),
+    url(r'^contributors/$', views.contributor_view),
+    url(r'^supporting/$', views.supporting_view),
+    url(r'^opposing/$', views.opposing_view),
```

Comment:
Not sure how you want to handle the `supporting`/`opposing` routes, but they should be relative to a `BallotItemSelection`. Supporters are in support of a Referendum OR Candidate.

So these routes should be more like `/measure/12/supporting` which returns a list of all the supporters for a Referendum with id 12. We should probably open a new issue to handle these routes as I suspect it's not quite that simple.

For this PR, it would be nice to at least take a referendum id or a candidate id so the frontend has something to work with.